### DISCUSSION TITLE
AzureMonitor: Resource picker search

### DISCUSSION THREAD

**Discussion by joshhunt** [grafana/grafana#41984] [Data Sources]
**What would you like to be added**:

 - Add a search field to resource picker to find resource groups and resources
 - Should be "good" search - case insensitive, fuzzy. Research ARG/KQL search operators.

**Comment by vickyyyyyyy** [comment]
Related to this: https://github.com/grafana/grafana/discussions/41809

**Comment by andresmgot** [comment]
The search feature is available in Grafana 9.0 after https://github.com/grafana/grafana/pull/48234
